drivers/python/rethinkdb/_import.py

```diff
@@ -515,9 +523,9 @@ def csv_reader(task_queue, filename, db, table, options, progress_info, exit_eve
 
 # This function is called through rdb_call_wrapper, which will reattempt if a connection
 # error occurs.  Progress will resume where it left off.
-def create_table(progress, conn, db, table, pkey, sindexes):
+def create_table(progress, conn, db, table, pkey, sindexes, shards, replicas):
     if table not in r.db(db).table_list().run(conn):
-        r.db(db).table_create(table, primary_key=pkey).run(conn)
+        r.db(db).table_create(table, primary_key=pkey, shards=shards, replicas=replicas).run(conn)
```

Comment:
I would suggest making a `create_args` dict directly in `options` rather than having both `shards` and `replicas`, something like:

```py
def parse_options():
  ...
  res['create_args'] = dict([(name, options[name]) for name in ['shards', 'replicas'] if getattr(options, name) != None])
  ...

def create_table(..., create_args):
  ...
    r.db(db).table_create(table, **dict(create_args.items() + [(primary_key, pkey)])).run(conn)
  ...
```

This could be made more readable if you're willing to break it into multiple lines.